reactive/src/main/scala/outwatch/reactive/Subscription.scala

```diff
@@ -0,0 +1,75 @@
+package outwatch.reactive
+
+import cats.Monoid
+
+import scala.scalajs.js
+
+sealed trait Subscription {
+  def cancel(): Unit
```

Comment:
I must admit that I prefer the name `Cancelable`. It is a bit more broad.

Reply:
Great Point! Subscription might be to specific. I even made an alias in `outwatch.dom` called `cancelable(() => ...)`.

I was not sure what to do with names and I am open to changes! I wanted to work in files that use `monix` and `outwatch.reactive` at the same time - having no name clashes made that easier. The same actually applies to `SinkSourceHandler`, `SourceStream` and `SinkObserver` - they could very well be named `Observable` and `Observer`. I even though about naming subscriptions `Cancellable`, but not sure.

What would you prefer?